RUFAS/routines/animal/life_cycle/calf.py

```diff
@@ -138,31 +138,49 @@ def calc_nutrient_rqmts(self, feed: Dict[str, float], temp: float) -> None:
         self.nutrient_rqmts = CalfRationManager.calc_requirements(self, feed, temp, self.animal_intake)
         self.DBW = self.nutrient_rqmts["live_weight_change"]["val"]
 
-    def calc_manure_excretion(self, feed: Dict[str, float], methane_model: str) -> None:
+    def calc_manure_excretion(
+        self, methane_model: str, nutrient_amount: Dict[str, float], nutrient_conc: Dict[str, float]
+    ) -> None:
         """
         Calculates and sets the manure excretion components.
 
         Parameters
         ----------
-        feed: Dict[str, float]
-            instance of the Feed class
         methane_model : str
-            methane model used for methane emission calculations
+            Methane model used for methane emission calculations, including Boadi, IPCC.
+        nutrient_amount : Dict[str, float]
+            Amount of nutrients in pen ration, calculated per animal, see Notes section for units.
+        nutrient_conc : Dict[str, float]
+            Concentration of nutrients in pen ration, calculated per animal, percentage.
```

Comment:
If you think this is unnecessary, please feel free to disregard. It's just a side effect of having a partner that's an English major.
```suggestion
            Concentrations of nutrients in pen ration, calculated per animal, percentage.
```